dashboard/app/controllers/levels_controller.rb

```diff
@@ -4,7 +4,7 @@
 class LevelsController < ApplicationController
   include LevelsHelper
   include ActiveSupport::Inflector
-  before_filter :authenticate_user!, :unless => Proc.new {params[:embed] && action_name == 'show'}
```

Comment:
hm, we use both embed_blocks and show with params[:embed]?

(@joshlory may be in the process of refactoring this controller to separate the level builder functionality from the normal user functionality)